### DISCUSSION TITLE
DICOM cast concept mappings

### DISCUSSION THREAD

**Discussion by ElliotSilver** [microsoft/dicom-server#1252] [General]
On the mappings page, you ask if ImagingStudy.modality should be derived from Modality (0008,0060) or Modalities in Study (0008,0061). ImagingStudy.modality is designed to be the equivalent of Modalities in Study. However, (0008,0061) will never appear in an instance; it is generated on-the-fly in a C-FIND response based on the union of the Modality (0008,0060) of the study's series. See [C.3.4. Additional Query/Retrieve Attributes ](http://dicom.nema.org/medical/dicom/current/output/html/part04.html#sect_C.3.4). Based on your use of dicom, your mapping is correct.

The comment below the tables about timestamps is slightly misleading. I would suggest the issue isn't storing the information in multiple elements, but in making those elements optional.  This is similar to how FHIR allows dates and datetimes to omit less significant digits.

You may find the (incomplete and informative) mappings found in FHIR STU3 [ImagingStudy mappings](http://www.hl7.org/fhir/stu3/imagingstudy-mappings.html) useful.

**Comment by StevenBorg** [comment]
@ElliotSilver, thank you so much for this Issue.  It has triggered a lot of conversation. We'll be updating our documentation periodically as we evolve DICOM Cast.  However, I'm going to leave this issue open while we address this issue, to provide a forum for a conversation.
